app/src/main/java/org/simple/clinic/forgotpin/createnewpin/ForgotPinCreateNewPinScreenKey.kt

```diff
@@ -6,7 +6,7 @@ import org.simple.clinic.R
 import org.simple.clinic.router.screen.FullScreenKey
 
 @Parcelize
-class ForgotPinCreateNewPinScreenKey : FullScreenKey {
+object ForgotPinCreateNewPinScreenKey : FullScreenKey {
```

Comment:
Same comment as before, we can convert this to a data class.

Reply:
Will make this change.